components/folder/ui-add-folder-button/AddFolderButton.jsx

```diff
@@ -0,0 +1,14 @@
+import styles from "@/components/folder/ui-add-folder-button/AddFolderButton.module.scss";
+import classNames from "classnames/bind";
+import { ReactComponent as AddIcon } from "@/components/folder/ui-add-folder-button/add.svg";
+
+const cx = classNames.bind(styles);
+
+export const AddFolderButton = ({ onClick }) => {
+  return (
+    <button className={cx("container")} onClick={onClick}>
+      <span>폴더 추가</span>
+      <AddIcon className={cx("icon")} />
```

Comment:
Next.js에서 svg 파일을 import할 때, CRA에서 컴포넌트로 가져올 수 있었던 것과 달리 아래 스크린샷처럼 객체로 가져오게 됩니다. 
<img width="443" alt="image" src="https://github.com/codeit-bootcamp-frontend/3-Weekly-Mission/assets/59413128/d468d433-41a3-457f-bc5a-5725ff6ea33f">
이것을 JSX 문법으로 컴포넌트화하려고 하니 잘못된 타입이라고 아래와 같은 에러가 발생하고 있습니다.
`Warning: React.jsx: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.`

이를 해결하는 방법 중 하나는  Next.js의 내장 컴포넌트인 Image 컴포넌트의 src 속성에 svg에서 import한 객체를 전달하는 것입니다.

```suggestion
import ADD_ICON_URL from "@/components/folder/ui-add-folder-button/add.svg";
import Image from "next/image";

const cx = classNames.bind(styles);

export const AddFolderButton = ({ onClick }) => {
	return (
		<button className={cx("container")} onClick={onClick}>
			<span>폴더 추가</span>
			<Image src={ADD_ICON_URL} alt='' className={cx("icon")} />
```

Reply:
그리고 이 방법 외에 CRA에서 했던 것처럼 SVG를 컴포넌트로 가져와서 사용하려면, svgr 라이브러리를 설치하여 사용하셔야 합니다.
방법은 링크를 첨부해드리니 참고 바랍니다.
[Nextjs에서 svg 사용하는 방법
](https://ui-log.github.io/docs/Nextjs/2022-11-24-Nextjs%EC%97%90%EC%84%9C-svg-%EC%82%AC%EC%9A%A9%ED%95%98%EB%8A%94-%EB%B0%A9%EB%B2%95/#%E1%84%89%E1%85%A5%E1%86%AF%E1%84%8C%E1%85%A5%E1%86%BC%E1%84%87%E1%85%A1%E1%86%BC%E1%84%87%E1%85%A5%E1%86%B8)